server/py/framework/db/sqldb/db.py

```diff
@@ -4798,7 +4807,9 @@ def _find_functions(
         """
         query = session.query(Function, Function.Tag.name)
 
-        if project != "*":
+        if isinstance(project, list):
+            query = query.filter(Function.project.in_(project))
+        elif project and project != "*":
```

Comment:
DRY, I think you can move to a base common func
`query = self._filter_project_resource(query, <resource>, project)`, whereas resource would be `X.project`
which will do exactly that